docs/serverless/alerts/view-alert-details.mdx

```diff
@@ -112,8 +112,7 @@ The Investigation section provides the following information:
     Add an <DocLink slug="/serverless/security/interactive-investigation-guides" section="add-investigation-guide-actions-to-a-rule">investigation guide</DocLink> to a rule when creating a new custom rule or modifying an existing custom rule's settings.
     </DocCallOut>
 
-* **Highlighted fields**: Shows relevant fields for the alert and any custom highlighted fields you added to the rule. 
-    {/* link to custom highlighted fields docs */}
+* **Highlighted fields**: Shows relevant fields for the alert and any custom highlighted fields you added to the rule. Custom highlighted fields with values are added to this section. Those without values aren't added.
```

Comment:
Same as previous comment, about the code comment and adding a link.

Reply:
```suggestion
* **Highlighted fields**: Shows relevant fields for the alert and any <DocLink slug="/serverless/security/rules-create" section="rule-ui-advanced-params"> custom highlighted fields</DocLink> you added to the rule. Custom highlighted fields with values are added to this section. Those without values aren't added.
```